deployment/setup.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Setup Raspberry Pi Base system
+  hosts: raspberries
+  gather_facts: yes
+  remote_user: pi
```

Comment:
I want to connect with a non-root-user, don't I?

Reply:
user pi is a non-root user with sudo privileges